src/anyp/Uri.cc

```diff
@@ -916,24 +914,25 @@ AnyP::Uri::Uri(AnyP::UriScheme const &aScheme) :
 char *
 AnyP::Uri::cleanup(const char *uri)
 {
-    int flags = 0;
     char *cleanedUri = nullptr;
     switch (Config.uri_whitespace) {
-    case URI_WHITESPACE_ALLOW:
-        flags |= RFC1738_ESCAPE_NOSPACE;
-    // fall through to next case
-    case URI_WHITESPACE_ENCODE:
-        flags |= RFC1738_ESCAPE_UNESCAPED;
+    case URI_WHITESPACE_ALLOW: {
+        const auto flags = RFC1738_ESCAPE_NOSPACE | RFC1738_ESCAPE_UNESCAPED;
         cleanedUri = xstrndup(rfc1738_do_escape(uri, flags), MAX_URL);
+    }
+    break;
```

Comment:
Just a more logical placement since the opening brace belongs to the entire `case`. I do not insist on this change.

```suggestion
        break;
    }
```